catchup/catchpointService.go

```diff
@@ -805,12 +807,27 @@ func (cs *CatchpointCatchupService) initDownloadPeerSelector() {
 	cs.blocksDownloadPeerSelector = cs.makeCatchpointPeerSelector()
 }
 
-func (cs *CatchpointCatchupService) makeCatchpointPeerSelector() *peerSelector {
-	return makePeerSelector(
-		cs.net,
-		[]peerClass{
-			{initialRank: peerRankInitialFirstPriority, peerClass: network.PeersPhonebookRelays},
-		})
+func (cs *CatchpointCatchupService) makeCatchpointPeerSelector() peerSelector {
```

Comment:
Back and forth - the specifics are for the `catchpointService` - I guess it could live in the `classBasedCatchpointPeerSelector` even if this is the only use of the constructor.